wordfreq/language_info.py

```diff
@@ -0,0 +1,150 @@
+from functools import lru_cache
+from langcodes import Language, best_match
+
+
+# Text in scripts written without spaces has to be handled specially in our
+# tokenization regex (see TOKEN_RE in tokens.py). Also, when one of these is
+# the script of the language we're analyzing, then we need to either have
+# a specific tokenizer for the language or give up.
+SPACELESS_SCRIPTS = [
+    # Han ideographs are spaceless, but they don't need to appear in this list
+    # because they have their own cases in get_language_info and TOKEN_RE.
+    'Hiragana',
+    # We omit katakana because Unicode regular expressions can already
+    # tokenize sequences of katakana, and omitting it here means we can also
+    # recognize a switch between hiragana and katakana as a token boundary.
+    'Thai',  # Thai script
+    'Khmr',  # Khmer script
+    'Laoo',  # Lao script
+    'Mymr',  # Burmese script
+    'Tale',  # Tai Le script
+    'Talu',  # Tai Lü script
+    'Lana',  # Lanna script
+]
+
+
+def _language_in_list(language, targets, min_score=80):
```

Comment:
`min_score` isn't used--not sure if it should be removed or if it should replace the `0` in line 35.